modal_proto/api.proto

```diff
@@ -1711,6 +1713,12 @@ message FunctionRetryPolicy {
   uint32 retries = 18;
 }
 
+message FunctionSchema {
+  int32 schema_version = 1; // 1 | 0 for now, allows easy disambiguation between empty schema and no schema collection
```

Comment:
Should this be an enum? I don't have a strong opinion... I mostly put it there to have a way to get field presence for schema to translate when moving the attribute to other protos